src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        String randomColor = String.valueOf(ColorSupplier.getRandomColor());
+        return new Ball(randomColor, randomNumber);
+    }
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

Reply:
Fix mistakes from the **checklist** in this class. There are 5 mistakes I think.